core/modules/admin_bar/css/admin_bar.css

```diff
@@ -83,6 +83,16 @@
   padding-right: 28px;
 }
 
+/* Back to Site arrow. */
+
+#admin-bar .dropdown li.admin-bar-icon > a.escape {
+  background-image: url("../images/chevron-disc-left.png");
```

Comment:
Now that the file names have changed, you need to change the file names here as well

Reply:
Now I make stupid mistakes.